install.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+#TODO(by black7375)
+#However, there may be existing user's settings, so back up and make a copy.
+#I just made a simple one yesterday, assuming that there may be multiple backup states.
+#I think it's better to do a redirection(>>) after backup to merge the contents of the file.
+printf "Are you using ESR, Default, or Dev?(ESR/Default/Dev): "
+read -r
+if [ "ESR" == "$REPLY" ] || [ "$REPLY" == "esr" ]; then
+  profiledir="$(grep '.default-esr' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+elif [ "Default" == "$REPLY" ] || [ "default" == "$REPLY" ]; then
+  profiledir="$(grep '.default-release' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+elif [ "Dev" == "$REPLY" ] || [ "dev" == "$REPLY" ]; then
+  profiledir="$(grep '.dev-edition-default' ~/.mozilla/firefox/profiles.ini | grep 'Default=' | cut -f 2 -d'=')"
+else
+  echo "Unspecified."
+  exit
+fi
+printf "Do you want to disable everything being selected when you click at the URL bar?(N/y): "
+read -r
+if [ "y" == "$REPLY" ] || [ "Y" == "$REPLY" ]; then
```

Comment:
https://stackoverflow.com/questions/226703/how-do-i-prompt-for-yes-no-cancel-input-in-a-linux-shell-script
It's generally better to use this approach.

Reply:
OK. I'll fix it.

Reply:
@black7375 If I do that, the lines get too long, greatly decreasing the readability. I think this will be a nofix.

Reply:
Then try using `2 space` as an indent.

Reply:
No I don't mea the indent. I mean using your method will make the lines longer and hard to read. Also using if instead of case is easier to add options. I guess it will be implemented next time.

Reply:
Well, it's a pretty standard way and it's not that hard.